dds_cli/options.py

```diff
@@ -225,6 +225,27 @@ def json_flag(help_message, long="--json", name="json", show_default=True):
     )
 
 
+def nomail_flag(
+    help_message,
+    long="--no-mail",
+    name="nomail",
```

Comment:
I think I want to change the name to `no_mail` instead. Just noting that here. Not important and nothing for you to change.